build/commonTest.props

```diff
@@ -18,11 +18,16 @@
     <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp2.1'">$(DotNetCoreAppRuntimeVersion)</RuntimeFrameworkVersion>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.1' OR '$(TargetFramework)' == 'net6.0' ">
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.1' OR '$(TargetFramework)' == 'net6.0' Or '$(TargetFramework)' == 'net8.0'">
```

Comment:
>$(TargetFramework)' == 'netcoreapp2.1' [](http://example.com/codeflow?start=30&length=38)

Why is netcoreapp2.1 still used?

Reply:
I don't remember any conversation about removing it. Should that be a different PR?

Reply:
@brentschmaltz what are your thoughts? I think a separate PR is fine, but let's at least open an issue for tracking

Reply:
Could it be to test the special case of aspnetcore2.1 on .NET FW (still maintained)

Reply:
nevermind. we discussed offline.